core-test/src/main/java/org/openstack4j/api/AbstractTest.java

```diff
@@ -71,6 +72,7 @@ private Service(int port) {
     protected static final String JSON_ACCESS = "/identity/v2/access.json";
     protected static final String JSON_TOKEN = "/identity/v3/authv3_project.json";
     protected static final String TOKEN_ID = "123456789";
+    protected static final Date DATE = new Date(1604096161000L);
```

Comment:
Do not pull this into `AbstractTest`. It is irrelevant for most of the implementations.